openedx/features/lti_course_tab/tab.py

```diff
@@ -34,7 +34,38 @@ class LtiCourseLaunchMixin:
     }
     DEFAULT_ROLE = 'Student'
 
+    def _get_pii_lti_parameters(self, course: CourseBlock, request: HttpRequest) -> Dict[str, str]:
+        """
+        Get LTI parameters that contain PII.
+
+        Args:
+            course (CourseBlock): CourseBlock object.
+            request (HttpRequest): Request object for view in which LTI will be embedded.
+
+        Returns:
+            Dictionary with LTI parameters containing PII.
+        """
+        lti_config = self._get_lti_config(course)
+        ask_to_send_username = lti_config.lti_config.get('ask_to_send_username', False)
```

Comment:
Are `ask_to_send_username` and `ask_to_send_email` two new fields we're adding to LtiConfiguration?  I'm wondering if we want these to be phrased more as authorizations or allowances.  `can_send_username` or something like that, perhaps?  I'm not sure if there's precedent for a naming scheme for this elsewhere in the codebase.

Reply:
@davidjoy These are the terms used by the [current LTI XBLock](https://github.com/edx/xblock-lti-consumer/blob/8f7c40f5a8a6465a3211692bf43978af5dc5fc38/lti_consumer/lti_xblock.py#L474-L487). I was going for familiarity here, but since they have different UIs perhaps it isn't that important. I will change this to what you suggested.

Reply:
Yeah, I'm not a huge fan of the `ask_to_` prefix; I initially thought this would prompt the user for consent.

That said, it may be even worse to refer to the variable by multiple different names.

Given the original key has existed in the xblock for >5 years now, I think I'd be begrudgingly inclined to leave it as-is :shrug:

Reply:
@stvstnfrd The field name is internal to the XBlock, and the display name is "Request user's username" the XBlock does actually ask for permission before sending this data. 

Perhaps we can use more accurate terminology internally while using familiar names in the UI?

Reply:
The current internal naming is to be consistent with the current XBlock field names. This might not be the best idea since these are used differently here. 
Perhaps the names should drop the ask and just be `send_username` and `send_email`.

Reply:
sure, that would clean up a little.